module/VuFind/src/VuFind/Db/Service/UserListServiceInterface.php

```diff
@@ -63,6 +63,26 @@ public function createEntity(): UserListEntityInterface;
      */
     public function getUserListById(int $id): UserListEntityInterface;
 
+    /**
+     * Get lists belonging to the user and their count. Returns an array of arrays with
+     * list_entity and count keys.
+     *
+     * @param int|UserEntityInterface $userOrId User entity object or ID
+     *
+     * @return array
+     * @throws Exception
+     */
+    public function getUserListsAndCountsByUser(int|UserEntityInterface $userOrId): array;
```

Comment:
This is a nitpick, but I needed some extra brain cycles here because the order in the documentation/variable name and type is different. 😅